mail.py

```diff
@@ -15,20 +16,44 @@
 name is ignored. Email is sent to uni@columbia.edu.
 """
 
-#ADDRESS used as the reply-to address
-ADDRESS = "cucs4118-tas@googlegroups.com"
+# Creates a CLI argument parser
+parser = argparse.ArgumentParser()
+
+parser.add_argument("filename",
+                    type=str,
+                    help="filename of the seating chart",
+                    metavar='<filename>')
+
+parser.add_argument("address",
+                    type=str,
+                    help="used as the reply-to address",
+                    metavar='<address>')
+
+parser.add_argument("subj",
+                    type=str,
+                    help="the message template subject name",
+                    metavar="<subj>")
+
+parser.add_argument("fromname",
+                    type=str,
+                    help="the name of the sender",
+                    metavar="<fromname>")
+
+parser.add_argument("fromaddr",
+                    type=str,
+                    help="the email address of the sender",
+                    metavar="<fromaddr>")
```

Comment:
This looks good overall - I think we should just change some of the argument names for clarity.
- address -> reply_to (but make the metavar "reply-to")
- subj -> subject
- fromname -> from_name (but make the metavar "from-name")
- fromaddr -> from_addr (but make the metavar "from-addr"

You'll have to change the relevant variables as well.